style/app.scss

```diff
@@ -792,6 +779,136 @@ div.sidebar.collapsed
   background-color : #dedede;
 }
 
+.focus-menu-bar
+{
+  position: fixed;
+  bottom: 0;
+  width: calc(100% - 50px);
+  margin-left: 25px;
+  margin-right: 25px;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.focus-menu-toggle
+{
+  position: fixed;
+  left: 25px;
+  border-radius: 17px 17px 0 0;
+  border: none;
+  background-color: #a273d3;
+  color: white;
+  padding: 7px 15px 0 15px;
+  font-size: 20px;
+  width: 150px;
+}
+
+.focuses-list
+{
+  display: inline-block;
```

Comment:
This class' style can be simplified; I tried disabling all three of these rules and the bar looked the same. So you can keep the class for later, but I don't think you need this block here.